pubspec.yaml

```diff
@@ -61,6 +65,8 @@ flutter:
     - assets/bluegradient.png
     - assets/loginpageimage.png
     - assets/bluelogo.png
+    - assets/bluelogo-nobg.png
```

Comment:
Use the same convention for naming, nowhere else is "-" used. Also, don't use abbreviation.

Reply:
Resolved